Content.Server/Access/Systems/IdCardConsoleSystem.cs

```diff
@@ -128,9 +128,9 @@ private void UpdateUserInterface(EntityUid uid, IdCardConsoleComponent component
         _idCard.TryChangeFullName(targetId, newFullName, player: player);
         _idCard.TryChangeJobTitle(targetId, newJobTitle, player: player);
 
-        if (_prototype.TryIndex<JobPrototype>(newJobProto, out var job)
-            && _prototype.TryIndex<StatusIconPrototype>(job.Icon, out var jobIcon))
+        if (_prototype.TryIndex<JobPrototype>(newJobProto, out var job))
         {
+            var jobIcon =  _prototype.Index(job.Icon);
```

Comment:
The changes here are all probably fine, and don't need to be reverted, but just note that even though it now uses the proto id structs & they should get linted, there's still no guarantee that a field doesn't somehow get set to an invalid value, or that an admin wont upload or VVs an invalid value.

So its generally still better to use TryIndex just in case, because depending on where it happens throwing exceptions could crash or freeze the game